backupninja/map.jinja

```diff
@@ -1,3 +1,4 @@
+{%- from "linux/map.jinja" import system with context %}

```

Comment:
I have parametrized meta/backupninja.ymk for contrail formula(check contrail formula PR) so when we load it service_grains jinja should be able to render it.